<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>405</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>285</width>
    <height>405</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Motor Control Relative</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="motorControl">
     <property name="minimumSize">
      <size>
       <width>261</width>
       <height>394</height>
      </size>
     </property>
     <property name="title">
      <string>Motor Control Relative</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="checkBox_enableArrows">
        <property name="text">
         <string>Move with arrow keys</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_same_xy">
        <property name="text">
         <string>Step [X] = Step [Y]</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="step_grid">
        <item row="2" column="0">
         <widget class="QLabel" name="label_step_y">
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>19</height>
           </size>
          </property>
          <property name="text">
           <string>Step [Y]</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>19</height>
           </size>
          </property>
          <property name="text">
           <string>Decimal</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spinBox_step_x">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>19</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_step_x">
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>19</height>
           </size>
          </property>
          <property name="text">
           <string>Step [X]</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spinBox_step_y">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>19</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>99.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_precision">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>19</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="direction_grid">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="1" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QToolButton" name="toolButton_up">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::UpArrow</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QToolButton" name="toolButton_down">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::DownArrow</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QToolButton" name="toolButton_left">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::LeftArrow</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <widget class="QToolButton" name="toolButton_right">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::RightArrow</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_stop">
        <property name="text">
         <string>Stop Movement</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
